arkh/assetlist.json

```
@@ -0,0 +1,31 @@
+{
+    "$schema": "../assetlist.schema.json",
+    "chain_id": "arkh",
+    "assets": [
+      {
+        "description": "The native token of Arkhadian",
+        "denom_units": [
+          {
+            "denom": "uarkh",
+            "exponent": 0,
+            "aliases": []
+          },
+          {
+            "denom": "arkh",
+            "exponent": 6,
+            "aliases": []
+          }
+        ],
+        "base": "uarkh",
+        "name": "Arkh",
+        "display": "arkh",
+        "symbol": "ARKH",
+        "logo_URIs": {
+          "png": "https://www.arkhadian.com/static/images/600-prism_ejAjDRh.jpg",
+          
+        },
+        "coingecko_id": "arkhadian"
```

```suggestion
        }
```

If not on coingecko yet, please don't add